hack/scripts/generate-cli-ginkgo-tests-summary.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+# Copyright 2023 VMware, Inc. All Rights Reserved.
```

Comment:
Did you write this script yourself or did you find it on the internet?
If it is taken from the internet, it is most probably fine, but we cannot put a VMware copyright

Reply:
I wrote the script.